docs/source/common/lightning_module.rst

```diff
@@ -983,6 +983,17 @@ The current logger being used (tensorboard or other supported logger)
         # the particular logger
         tensorboard_logger = self.logger.experiment
 
+loggers
+~~~~~~~
+
+The list of loggers currently being used.
+
+.. code-block:: python
+
+    def training_step(self, batch, batch_idx):
+        # List of LightningLoggerBase objects
```

Comment:
I don't believe this example provides enough information.

Reply:
```py
from pytorch_lightning.loggers import TensorBoardLogger, WandbLogger


class Model(LightningModule):

    def training_step(self):
        tensorboard_logger, wandb_logger = self.loggers
        ...


save_dir = './logs'
trainer = Trainer(logger=[TensorBoardLogger(save_dir), WandbLogger(save_dir)])
```

Reply:
I updated the example with this suggestion https://github.com/PyTorchLightning/pytorch-lightning/pull/11683#discussion_r802237522 which is the same example used in trainer.rst. Let me know if you still think it needs to be updated.